packages/stream/solana/StreamClient.ts

```diff
@@ -849,6 +1020,32 @@ export class SolanaStreamClient extends BaseStreamClient {
     return new Contract(decodeStream(escrow?.data));
   }
 
+  /**
+   * Fetch all aligned outgoing streams/contracts by the provided public key.
+   */
+  public async getOutgoingAligned(publicKey: PublicKey): Promise<Record<string, Stream>> {
```

Comment:
Could we have more descriptive signature and/or the comment above? 

Do consumers need to call it after `get` for distinguishing aligned and regular streams?

Reply:
hmm, right now `get` calls this function to get the aligned streams, so this is more of a helper function. I would say it makes more sense to make this private, but create a way to fetch only aligned/non-aligned streams